.buildkite/hooks/post-checkout

```diff
@@ -0,0 +1,13 @@
+# Freshen the remote so that our diff is accurate
+git fetch origin
+./.buildkite/scripts/generate-diff.sh > $computed_diff.txt
+
+#
+# TODO: Remove this when we figure out where to stick this cache long term
```

Comment:
You can pre-bake the cache into the container. All you need to do is just do `dhall <<< '<external_lib_wrapper_path>'` for each external library and the cache will be there for you. You will just need to re-build the container when you update dhall deps, which should be very, very infrequent. We can add a update_deps style step for enforcing that if we need to.

Reply:
oops, you're right. This is a simple way to do it